container/default.json

```
@@ -128,21 +128,21 @@
   "footer": {
     "links": {
       "text": "<0>ISIS Home</0> | <2>Privacy statement</2> | <4>Data policy</4> | <6>Accessibility statement</6> | <8>Contact</8>",
-      "facility": "#",
-      "privacy-statement": "#",
-      "data-policy": "#",
-      "contact": "#"
+      "facility": "https://www.isis.stfc.ac.uk/Pages/home.aspx",
+      "privacy-statement": "/cookies",
+      "data-policy": "https://www.isis.stfc.ac.uk/Pages/Data-Policy.aspx",
+      "contact": "mailto:fia@stfc.ac.uk?subject=Question%20about%20FIA"
     },
     "website-development-provider": "Built by the <1>Scientific Computing</1> <2>Data and Software Engineering Group</2>"
   },
   "navigation-drawer": {
     "alternative-text": "Science and Technology Facilities Council"
   },
   "accessibility-page": {
-    "title": "Accessibility statement for DataGateway",
-    "domains-list-text": "This accessibility statement applies to all facilities using DataGateway. This includes:",
-    "domains-list": ["https://data.isis.stfc.ac.uk "],
-    "website-developed-by": "These sites are maintained by the Data Software and Engineering Group, in the Scientific Computing Department. ",
+    "title": "Accessibility statement for FIA",
+    "domains-list-text": "This accessibility statement applies to all facilities using FIA. This includes:",
+    "domains-list": ["fia"],
```

```suggestion
    "domains-list": ["reduce.isis.cclrc.ac.uk"],
```
Probably want to use this here